modules/travel_pay/spec/controllers/claims_controller_spec.rb

```diff
@@ -5,62 +5,36 @@
 RSpec.describe TravelPay::ClaimsController, type: :request do
   let(:user) { build(:user) }
 
+  before do
+    sign_in(user)
+  end
+
   describe '#index' do
     context 'successful response from API' do
-      it 'responds with 200' do
-        allow_any_instance_of(TravelPay::Client)
-          .to receive(:request_veis_token)
-          .and_return('veis_token')
-
-        allow_any_instance_of(TravelPay::Client)
-          .to receive(:request_sts_token)
-          .and_return('sts_token')
-
-        allow_any_instance_of(TravelPay::Client)
-          .to receive(:request_btsss_token)
-          .with('veis_token', 'sts_token')
-          .and_return('btsss_token')
-
-        allow_any_instance_of(TravelPay::Client)
-          .to receive(:get_claims).and_return([])
-
-        sign_in(user)
-
-        get '/travel_pay/claims', params: nil, headers: { 'Authorization' => 'Bearer vagov_token' }
+      let(:expected_claim_ids) do
+        %w[
+          6ea23179-e87c-44ae-a20a-f31fb2c132fb
+          6ea23179-e87c-44ae-a20a-f31fb2c021fb
+          6ea23179-e87c-44ae-a20a-f31fb2c782fb
```

Comment:
Nit pick, but these should probably all be different :)

Reply:
They are different! But updated to make it clearer